addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -95,6 +96,7 @@ export class SuggestionService {
     }
 
     searchPartnerSuggestions(cleanedSearchTerm, thread, sort) {
+        thread = toRaw(thread);
```

Comment:
We could `toRaw()` the record(s) even earlier, in `searchSuggestions()`.

Reply:
watch out that `thread` is optional in `searchSuggestions`, don't think `toRaw(undefined)` works

Reply:
It works :)

Reply:
It works :)

Reply:
Indeed, from how this is implemented. My fear comes from `reactive(undefined)` not working but `toRaw(val)` returns `val` when this doesn't map to a reactive raw value and since no reactive can have raw value `undefined` then it's fine and return `undefined` anyway :)